docs/binary_format/binaryFormat.md

```diff
@@ -0,0 +1,154 @@
+
+# Typedefs binary format
+
+Terms are serialised in a type-directed way, given a typedefs
+description --- terms make no sense on their own. If the type is
+unknown, a description of the type could first be serialised, followed
+by the serialisation of the term (this is future work).
```

Comment:
```suggestion
by the serialisation of the term. (This is future work.)
```

Reply:
It is not clear to me why this change is necessary. Could you explain your reasoning to me?

Reply:
In this case, it's more or less that the sentence still has to make sense without parens. Alternatively, emdash (or endash, I always forget which is which) could work, e.g. "-- this is future work.", or "(which is future work)."

Reply:
Oh, I just noticed you do use `---` in other places.

Reply:
I think the `---` is from fred but I'll put it here as well.